pkg/kubelet/dockershim/cm/container_manager_linux.go

```diff
@@ -123,8 +123,9 @@ func createCgroupManager(name string) (cgroups.Manager, error) {
 		Parent: "/",
 		Name:   name,
 		Resources: &configs.Resources{
-			Memory:     int64(memoryLimit),
-			MemorySwap: -1,
+			Memory:      int64(memoryLimit),
+			MemorySwap:  -1,
+			SkipDevices: true,
 			Devices: []*configs.DeviceRule{
```

Comment:
@giuseppe I think that `Devices: ` entry can be `nil` now, so makes sense to remove it from here to avoid confusion.

Reply:
same nit in the other two places.

Reply:
it seems to be still used to setup the systemd service.  If I drop the `Devices` entry then all devices are blocked.

Reply:
Ah, that is a bug in runc :( I'll follow up on that one.

Reply:
Better late than never, https://github.com/opencontainers/runc/pull/2958 🤡